packages/hadron-express/index.ts

```diff
@@ -16,11 +19,22 @@ import {
 
 export default hadronExpress;
 
-export const register = (container: IContainer, config: IHadronExpressConfig) =>
-  hadronExpress(
+export const register = (
+  container: IContainer,
+  config: IHadronExpressConfig,
+) => {
+  const port = process.env.PORT || 8080;
```

Comment:
I think we should make a possibility to change the default port in the config object.